src/reader/ArticleReader.js

```diff
@@ -124,7 +107,13 @@ export default function ArticleReader({ api, teacherArticleID }) {
   };
 
   const handleScroll = () => {
-    let ratio = getScrollRatio();
+    let bottomRowHeight = document.getElementById("bottomRow");
```

Comment:
this is the `bottomRow` not the height of it. the name is misleading. consider extracting to helperMethod?